src/sentry/models/group.py

```diff
@@ -379,6 +379,37 @@ def get_tags(self, with_internal=True):
 
         return self._tag_cache
 
+    def get_first_release(self):
+        from sentry.models import GroupTagValue
+
+        # find first seen release
```

Comment:
FYI these comments are useless. This is literally the name of the function. So this comment isn't adding more value. :)